src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
+
+  type Key = Array[Byte]
+  type Value = Array[Byte]
```

Comment:
We use `ByteString`s almost everywhere else in the system. Imagine a situation when a new storage would use `ByteString`s as well. Then we would have to convert whole data passing through a `DataSource` to `Array[Byte]` and back to `ByteString`.